server/views/components/assessment/_assessment-area.njk

```diff
@@ -29,20 +29,26 @@
                 {% if assessment.criminogenicNeedMissing %}
                     <p>This information isn't currently available, please check in OASys.</p>
                 {% else %}
-                    {% if linkedtoRoSH or riskOfSeriousHarmData %}
+                    {% if linkedtoRoSH %}
                         <div class="rosh">
                             <h3 class="govuk-heading-s">{{ locale.detail.linkedToRoSH }}</h3>
                             <p class="govuk-body">{{ riskOfSeriousHarmData }}</p>
                         </div>
+                    {% elseif riskOfSeriousHarmData  %}
+                        <h3 class="govuk-heading-s">{{ locale.detail.notLinkedToRoSH }}</h3>
+                        <p class="govuk-body">{{ riskOfSeriousHarmData }}</p>
                     {% else %}
                         <h3 class="govuk-heading-s">{{ locale.detail.notLinkedToRoSH }}</h3>
                     {% endif %}
 
-                    {% if linkedtoReoffending or riskOfReoffendingData %}
+                    {% if linkedtoReoffending %}
                         <div class="ror">
                             <h3 class="govuk-heading-s">{{ locale.detail.linkedToReoffending }}</h3>
                             <p class="sp-text-pre-wrap govuk-body">{{ riskOfReoffendingData }}</p>
                         </div>
+                    {% elif riskOfReoffendingData %}
```

Comment:
above you use elseif - I know they're the same but we should prefer one or the other in the same file I think.

Reply:
Good spot! I've changed it to 'elseif'